content/en/docs/concepts/architecture/cloud-controller.md

```diff
@@ -231,6 +231,7 @@ The following cloud providers have implemented CCMs:
 * [DigitalOcean](https://github.com/digitalocean/digitalocean-cloud-controller-manager)
 * [GCP](https://github.com/kubernetes/cloud-provider-gcp)
 * [Hetzner](https://github.com/hetznercloud/hcloud-cloud-controller-manager)
+* [HUAWEI CLOUD](https://github.com/kubernetes-sigs/cloud-provider-huaweicloud)
```

Comment:
Should the link text be "HUAWEI Cloud" based on branding here: https://cloud.huawei.com/?